taxonomies/ingredients.txt

```diff
@@ -47146,6 +47214,10 @@ it:nocciole tritate
 openfoodfacts:https://world.openfoodfacts.org/ingredient/fr:noisettes-hachées
 # 44 products @2018-10-07
 
+<en:chopped hazelnuts
+en:chopped toasted hazelnuts
+es:avellana picada y tostadas
```

Comment:
we should add "picadas" to the ingredients processing taxonomy instead

Reply:
ok